# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/dosemu/hotfix-debuglevel.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- dosemu-1.4.0/src/base/async/dyndeb.c.vanilla	2025-05-08 22:31:13.409818904 +0100
+++ dosemu-1.4.0/src/base/async/dyndeb.c	2025-05-08 22:33:52.469756037 +0100
@@ -71,6 +71,7 @@
 	return 0;
 }
 
+#if 0
 int debug_level(int letter)
 {
 	struct debug_class *class;
@@ -83,6 +84,7 @@
 	}
 	return class->level;
 }
+#endif
 
 
 /*
--- dosemu-1.4.0/src/include/dosemu_debug.h.vanilla	2025-05-08 22:36:30.424021717 +0100
+++ dosemu-1.4.0/src/include/dosemu_debug.h	2025-05-08 22:36:42.371355167 +0100
@@ -179,8 +179,8 @@
 extern int unregister_debug_class(int letter);
 extern void print_debug_usage(FILE *stream);
 extern int set_debug_level(int letter, int level);
-extern inline int debug_level(int letter);
-extern inline int debug_level(int letter)
+inline int debug_level(int letter);
+inline int debug_level(int letter)
 {
 	if (letter >= DEBUG_CLASSES) {
 		return -1;
